android/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: "com.android.library"
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 26
```

Comment:
This value should be set to the latest SDK which is currently 28.

Reply:
> August 2019: New apps are required to target API level 28 (Android 9) or higher.
> November 2019: Updates to existing apps are required to target API level 28 or higher.

https://android-developers.googleblog.com/2019/03/introducing-android-q-beta.html